system/lib/libcxxabi/src/cxa_exception_emscripten.cpp

```diff
@@ -45,19 +45,27 @@ char* emscripten_format_exception(void* thrown_object) {
     const char* what =
       static_cast<const std::exception*>(thrown_object)->what();
     asprintf(&result,
-             "terminating with uncaught exception of type %s: %s",
+             (terminate ? "terminating with uncaught exception of type %s: %s"
+                        : "exception of type %s: %s"),
              type_name,
              what);
   } else {
-    asprintf(
-      &result, "terminating with uncaught exception of type %s", type_name);
+    asprintf(&result,
+             (terminate ? "terminating with uncaught exception of type %s"
+                        : "exception of type %s"),
+             type_name);
   }
 
   if (demangled_buf) {
     free(demangled_buf);
   }
   return result;
 }
+
+char* __get_exception_terminate_message(void *thrown_object) {
```

Comment:
I guess we don't have any current caller of this yet, but we will do soon (perhaps in debug builds, or some opt-in configuration)?

Reply:
Yes I'd like to use it later, and this PR doesn't have a direct test for this, but I think holding this small piece off is probably not worth it..